data/drak/indigenous.txt

```diff
@@ -298,21 +299,7 @@ ship "Embersylph"
 	turret 0 0 "Absorption Organ"
 	leak "blood" 2
 	explode "blood" 1
-	description "Ember Ray."
-
-
-
-ship "Embersylph" "Embersylph (Small)"
-	sprite "ship/embersylph/embersylph"
-		"frame rate" 36
-		"random start frame"
-		"scale" 0.5
-	"never disabled"
-	add attributes
-		"shields" -2800
-		"hull" -900
-		"thrust" 2
-		"turn" 10
+	description "Embersylphs are strange creatures that glow in various hues, gracefully swinging several long appendiges in a strangely methodic movement. The only known behavior these creatures are known to display is an act of curiosity for other life."
```

Comment:
```suggestion
	description "Embersylphs are strange creatures that glow in various hues, gracefully swinging several long appendages in a strangely methodic movement. The only consistent behavior these creatures are known to display is curiosity for other life."
```

Minimum recommendation, removing redundancy and the odd "act of curiosity" phrasing in addition to spelling fix.